bin/tx-prover/src/proxy/mod.rs

```diff
@@ -31,15 +37,77 @@ const LOCALHOST_ADDR: &str = "127.0.0.1";
 // WORKER
 // ================================================================================================
```

Comment:
This file is getting pretty big. I would probably move the `Worker` struct into a separate file - e.g., `src/proxy/worker.rs`.

Reply:
Makes sense. Changing it.